npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -800,7 +800,7 @@ def restore_packages_name(updated_lockfile_content, parsed_updated_lockfile_cont
 
         sig do
           params(
-            current_name: String,
+            current_name: T.any(String, NilClass),
```

Comment:
Can you describe the scenario that results in this being nil? 

Just want to ensure this isn't hiding another bug elsewhere in the code that leads into this...

Reply:
I tried to reproduce it through existing public repositories which had error on sentry. But all of them are passing. Only I can see a common thing for them that they don't have `name` in `package.json`. But currently they do have name in `package-lock.json` file. So I think for the first time when dependabot runs and try to update locker file there is no name but later they have name then it is working fine.

Reply:
Bofore calling this method we are also checking original_name biut not the current_name.